hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -508,6 +512,21 @@ signWith (TxId h) (PaymentVerificationKey vk, PaymentSigningKey sk) =
     Ledger.Shelley.WitVKey
       (Ledger.asWitness vk)
       (Ledger.signedDSIGN @Ledger.StandardCrypto sk h)
+-- * Extra
+
+-- TODO: Could be offered upstream.
+class UseForBuildingTx f where
+  useForBuildingTx :: f CtxUTxO Era -> f CtxTx Era
```

Comment:
I was confused by the name first and wondered what it's doing. Maybe it's missing "Context" in the name as it's re-purposing / re-labeling of things which are in `CtxUTxO` to `CtxTx`? As a user of `TxOut ctx era` I would know the concept of context. What about naming this: `toTxContext` (and `ToTxContext`) or so?

Reply:
It was named 'toCtxTx' initially but as Arnaud pointed out in an earlier comment, this was a bit cryptic so I went for something which IMO better convey what really is this transformation about (rather than, what it does to the type).

Reply:
Changed to `toTxContext` in: 024b6ea